pages/projects/projects-spacetimecube.tsx

```diff
@@ -102,6 +121,46 @@ const ProjectSpaceTimeCube: NextPage<Props> = ({
       });
     },
   );
+  const options = {
```

Comment:
Perhaps with horizontal bars it would be easier to accomodate more and longer labels? (Ideally, we would use the country name's instead of the iso codes)

Reply:
Ok, I will try to do it!